docs/sources/alerting/manage-notifications/view-alert-rules.md

```diff
@@ -40,8 +40,8 @@ When managing large volumes of alerts, you can use extended alert rule search ca
 
 To view alerting details:
 
-1. In the Grafana menu, click the **Alerting** (bell) icon to open the Alerting page. By default, the List view displays.
-1. In **View as**, toggle between Grouped or State views by clicking the relevant option. See [Grouped view](#grouped-view) and [State view](#state-view) for more information.
+1. Click **Alerts & IRM** -> **Alert rules**. By default, the List view displays.
```

Comment:
I _think_ the title of this section changes to just "alerting" if you don't have either Incident or OnCall installed, do we care about that?

Reply:
oh does it? hmm let me think